wpilibNewCommands/src/main/native/include/frc2/command/Subsystem.h

```diff
@@ -83,6 +83,13 @@ class Subsystem {
    */
   void SetDefaultCommand(CommandPtr&& defaultCommand);
 
+  /**
+   * Removes the default command for the subsystem.  The current default command
```

Comment:
I'm not sure I like the latter part of this comment, seems easy to think that calling this will schedule the default command.

Reply:
would a better rewording be "The current default command will not be canceled automatically"?

Reply:
Perhaps "this will not cancel the default command if it is currently running"?